iot-hub/Quickstarts/read-d2c-messages/ReadDeviceToCloudMessages.js

```diff
@@ -1,55 +1,87 @@
+  for more details at https://www.npmjs.com/package/@azure/event-hubs
+
+  For an example that uses checkpointing, follow up this sample with the sample in the 
+  eventhubs-checkpointstore-blob package on GitHub at the following link:
+
+  https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/eventhubs-checkpointstore-blob/samples/javascript/receiveEventsUsingCheckpointStore.js
+*/
+
+const { EventHubConsumerClient } = require("@azure/event-hubs");
+
+// If using websockets, uncomment the below require statement
+// const WebSocket = require("ws");
+
+// If you need proxy support, uncomment the below code to create proxy agent
+// const HttpsProxyAgent = require("https-proxy-agent");
+// const proxyAgent = new HttpsProxyAgent(proxyInfo);
 
 var printError = function (err) {
   console.log(err.message);
 };
 
 // Display the message content - telemetry and properties.
 // - Telemetry is sent in the message body
-// - The device can add arbitrary application properties to the message
+// - The device can add arbitrary properties to the message
 // - IoT Hub adds system properties, such as Device Id, to the message.
-var printMessage = function (message) {
-  console.log('Telemetry received: ');
-  console.log(JSON.stringify(message.body));
-  console.log('Application properties (set by device): ')
-  console.log(JSON.stringify(message.applicationProperties));
-  console.log('System properties (set by IoT Hub): ')
-  console.log(JSON.stringify(message.annotations));
-  console.log('');
+var printMessages = function (messages) {
+  for (const message of messages) {
+    console.log("Telemetry received: ");
+    console.log(JSON.stringify(message.body));
+    console.log("Properties (set by device): ");
+    console.log(JSON.stringify(message.properties));
+    console.log("System properties (set by IoT Hub): ");
+    console.log(JSON.stringify(message.systemProperties));
+    console.log("");
+  }
 };
 
-// Connect to the partitions on the IoT Hub's Event Hubs-compatible endpoint.
-// This example only reads messages sent after this application started.
-var ehClient;
-EventHubClient.createFromIotHubConnectionString(connectionString).then(function (client) {
-  console.log("Successfully created the EventHub Client from iothub connection string.");
-  ehClient = client;
-  return ehClient.getPartitionIds();
-}).then(function (ids) {
-  console.log("The partition ids are: ", ids);
-  return ids.map(function (id) {
-    return ehClient.receive(id, printMessage, printError, { eventPosition: EventPosition.fromEnqueuedTime(Date.now()) });
+async function main() {
+  console.log("IoT Hub Quickstarts - Read device to cloud messages.");
+
+  // Event Hub-compatible endpoint
+  // az iot hub show --query properties.eventHubEndpoints.events.endpoint --name {your IoT Hub name}
+  const s_eventHubsCompatibleEndpoint = "{your Event Hubs compatible endpoint}";
```

Comment:
Nit 1: Is there anyway to move these const statements to the top of the file? If that's bad form never mind or if you'd rather keep them here that's fine too. (They won't be hard to find.)

Nit 2: are the constant names compatible with Node naming conventions? (I don't know, I'm just asking.) I can always change the doc page if they aren't and you rename them.

PS: both of these are NITS. Totally up to you wrt what is best.

Reply:
Yes, they can be moved up, not a problem.
No, the names dont follow the Node naming conventions :( We like camelCasing
I followed this as you asked to match your quick start update.

What does `s_` even mean??

Reply:
Cool. If you would move them up, that would be great. I have no idea what "s_" means, we can probably drop that :)  Feel free to change the constants to whatever casing and names make sense and I'll change the docs to match.

Reply:
addressed in https://github.com/Azure-Samples/azure-iot-samples-node/pull/39/commits/058d62c74ce7afa6965a8dc09eced55af9e4a340